src/lib/components/submit/SubmitMotion.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+    import type { ProposalContent } from '$lib/types/juno';
+    import type { ProposalEditableMetadata } from '$lib/types/juno';
+    import SubmitWriteEdit from '$lib/components/submit/SubmitWriteEdit.svelte';
+    import { getEditable } from '$lib/services/idb.services';
+    import { onMount } from 'svelte';
+
+    let metadata: ProposalEditableMetadata | undefined;
+    let content: ProposalContent | undefined;
+
+    onMount(async () => ([metadata, content] = await getEditable()));
```

Comment:
This should not be moved here but remain in its original component because it's important to load those information for the wizard flow.